id.go

```diff
@@ -14,8 +14,14 @@ import (
 const (
 	hexDigits     = "0123456789abcdefABCDEF"
 	macDelimiters = ":-.,"
-	macPrefix     = "mac"
 	macLength     = 12
+
+	SchemeMAC    = "mac"
```

Comment:
Nicely done on the SchemeXXX exported fields.  We needed to have done that for a while.

We should probably reserve a value for an "unknown" or "missing" schema.  `SchemeUnknown` probably works.